lib/lita/handlers/lunch_reminder.rb

```diff
@@ -161,6 +165,9 @@ def load_on_start(_payload)
           "@#{giver.mention_name}, le has dado uno de tus puntos de " +
           "karma a @#{destinatary.mention_name}."
         )
+        broadcast_to_audit_channel("@#{giver.mention_name}, le ha dado un punto de " +
+          "karma a @#{destinatary.mention_name}."
+        )
```

Comment:
Closing method call brace must be on the same line as the last argument when opening brace is on the same line as the first argument.